src/test/java/com/stripe/model/DeserializerTest.java

```diff
@@ -105,6 +105,14 @@ public void deserializeAppFeeRefundList() throws IOException {
 	}
 
 	@Test
+	public void deserializeAppFee() throws IOException {
+		String json = resource("appfee_refund_list.json");
```

Comment:
I can grab this on a followup PR but I'd love to rename this to "appfee_with_refund_list.json" because my first thought was "what?  a list of Application Fee Refunds?" when really it's one applicaiton fee object, that has a list of refunds.